.github/workflows/broken-linkcheck.yml

```diff
@@ -0,0 +1,34 @@
+name: Linkcheck
+on:
+    workflow_dispatch: # Enable manual triggering
+    #schedule:
+    # Run monthly on the 1st day of the month
+        #- cron: '0 0 1 * *'
+    pull_request:
+      paths:
+        - ".github/workflows/broken-linkcheck.yml"
+        - "doc/make.py"
+jobs:
+    linkcheck:
+        runs-on: ubuntu-latest
+
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+                fetch-depth: 0
+
+        - name: Setup Python
+          id: setup_python
+          uses: actions/setup-python@v4
+          with:
+            python-version: '3.10'
+            cache: 'pip'
+            cache-dependency-path: 'requirements-dev.txt'
+
+        - name: Install requirements-dev.txt
+          run: pip install -r requirements-dev.txt
```

Comment:
CI is failing, because you are missing the `pandas` installation step. For example

https://github.com/pandas-dev/pandas/blob/d01669f5cdfff4edc889994331901b1f97946fdd/.github/workflows/docbuild-and-upload.yml#L43-L47